CHANGES.md

```diff
@@ -154,36 +149,35 @@ now run in untrusted environment, with limitations.  Applets cannot
 create classloaders, so BeanShell cannot script classes in the unsigned
 environment.  These demos are very rough and need some work.
 
-- The show() command now works in both interactive and non-interactive
+- The `show()` command now works in both interactive and non-interactive
 modes.  This means that for experimentation, instead of doing things like:
-```java
-print( myObject instanceof Blah );
-print( myObject.getFoo() );
-print( myObject.somethingElse() );
-```
+    ```java
+    print( myObject instanceof Blah );
+    print( myObject.getFoo() );
+    print( myObject.somethingElse() );
+    ```
```

Comment:
Indenting fenced code blocks is *really* not necessary, and IMO harder to maintain when editing the file.

Might also want a blank line before and after the fenced block for readability of the raw markdown.

Noticed this throughout the file.

Reply:
This code block is part of a bulleted list element. All content belonging to a list element must also be indented to the same level as the first line of the bulleted element. If you don't indent it, it's no longer part of the list, which changes how markdown renders it (at least in GitHub).

Reply:
Also fair. somehow, these *look* like they are indented further than the list-element, though.

Reply:
The code block and text in this list item were not all evenly indented. Fixed here: [6c9610c](https://github.com/beanshell/beanshell/pull/641/commits/6c9610c84923d3e87b178f6c54b0ec1ef4aa7994)

This doesn't affect rendering but does make it easier to read the raw Markdown.

Reply:
@Sailsman63 Did my last change to the indentation ([6c9610c](https://github.com/beanshell/beanshell/pull/641/commits/6c9610c84923d3e87b178f6c54b0ec1ef4aa7994)) fix the indentation issue you saw?

Reply:
Sorry, I meant to reply earlier. Yes, this looks good.